src/components/EmbeddedComponent.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { IframeHTMLAttributes } from 'react';
+import { useCustomerLink } from 'hooks/useCustomerLink';
+
+type IFrameProps = Omit<IframeHTMLAttributes<HTMLIFrameElement>, 'src'>;
+type Props = {
+  apiKey: string;
+  customerName: string;
+  baseApiUrl?: string;
+} & IFrameProps;
+
+export const EmbeddedComponent = (props: Props) => {
+  const { apiKey, customerName, ...iframeProps } = props;
+  const { result } = useCustomerLink({
+    apiKey,
+    customerName,
+  });
+
+  if (!result?.url) {
+    return null;
+  }
+
+  return <iframe src={`https://${result?.url}`} {...iframeProps} />;
```

Comment:
The customer's link endpoint currently responses with a link w/o a protocol - that's why I had to add `https://`

Reply:
That seems like something we may want to fix on the backend. I'm not sure why we wouldn't include the protocol (we only serve it over https). In the case that we fix this endpoint, you may want to check if the URL has a protocol, and only prepend `https://` if it's missing.